app/graphql/types/invoice_subscription/object.rb

```diff
@@ -15,6 +15,8 @@ class Object < Types::BaseObject
       field :fees, [Types::Fees::Object], null: true
       field :from_datetime, GraphQL::Types::ISO8601DateTime, null: true
       field :to_datetime, GraphQL::Types::ISO8601DateTime, null: true
+      field :charges_from_datetime, GraphQL::Types::ISO8601DateTime, null: true
```

Comment:
GraphQL/OrderedFields: Fields should be sorted in an alphabetical order within their section. Field `charges_from_datetime` should appear before `to_datetime`.